packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -1,20 +1,76 @@
+  "attribute.time": "Hora",
+  "attribute.timestamp": "Sello de hora",
+  "attribute.uid": "UID",
+  "attribute.uid.description": "Indentificador único",
+  "button.attributes.add.another": "Agregar otro campo",
+  "button.component.add": "Agregar un componente",
+  "button.component.create": "Crear nuevo componente",
+  "button.model.create": "Crear nueva Colección",
+  "button.single-types.create": "Crear nuevo contenedor único",
+  "component.repeatable": "(repetible)",
+  "components.componentSelect.no-component-available": "Ya agregaste todos tus componentes",
+  "components.componentSelect.no-component-available.with-search": "No se encontró ningún componente para tu búsqueda",
+  "components.componentSelect.value-component": "un componente seleccionado (escribe para buscar uno distinto)",
+  "components.componentSelect.value-components": "{number} componentes seleccionados",
+  "configurations": "Configuración",
+  "contentType.collectionName.description": "Útil cuando el nombre de tu Contenedor y el nombre de tu tabla difieren",
+  "contentType.collectionName.label": "Nombre de la Colección",
+  "contentType.displayName.label": "Nombre desplegado",
+  "contentType.kind.change.warning": "Cambiaste el contenido de tu Contenedor: La API se reiniciará (rutas, controladores, servicios se reescribirán).",
+  "contentType.UID.description": "El UID se usa para generar las rutas de la API y las tablas/colecciones de la base de datos",
+  "error.contentTypeName.reserved-name": "Este nombre no puede ser usado en tu proyecto, puede interferir otras funciones",
+  "error.validation.enum-duplicate": "Valores duplicados no son permitidos",
+  "error.validation.minSupMax": "No puede ser mayor al máximo",
+  "error.validation.minSupMax": "El patrón Regex no es válido",
+  "error.validation.relation.targetAttribute-taken": "Este nombre ya esiste",
+  "form.attribute.component.option.add": "Agregar un componente",
+  "form.attribute.component.option.create": "Crear un nuevo componente",
+  "form.attribute.component.option.create.description": "Un componente puede compartir con Contenedores y Componentes, estará disponible y accesible en todos lados.",
+  "form.attribute.component.option.repeatable": "Componente repetible",
+  "form.attribute.component.option.repeatable.description": "Mejor para Instancias múltiples (arreglo) de ingredientes, meta etiquetas, etc..",
+  "form.attribute.component.option.reuse-existing": "Usar un componente existente",
+  "form.attribute.component.option.reuse-existing.description": "Reusar un componente ya creado mantiene la consistencia de tus datos entre Contenedores",
+  "form.attribute.component.option.single": "Componente simple",
+  "form.attribute.component.option.single.description": "Mejor para agrupar campos, como domicilios completos, información principal, etc...",
   "form.attribute.item.customColumnName": "Nombres de columna personalizados",
```

Comment:
I'm not sure if this key refers tu plural or singular? If it's referring to singular, it should be "nombre de columna personalizado"

Reply:
I think it is singular. "Nombre de columna personalizable" will be much clear in terms of grammar